src/components/social/facebook/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
```

Comment:
Why are you importing the store here, a better approach would be using react-redux and import the actions which you want to access